src/app/content/highlights/components/SummaryPopup/__snapshots__/ChapterFilter.spec.tsx.snap

```diff
@@ -248,7 +249,7 @@ input:checked + .c10 .c11 {
         className="c7 c8"
       >
         <input
-          checked={true}
+          checked={false}
```

Comment:
this seems worrying

Reply:
I've added only `hook(addCurrentPageToSummaryFilters());` to those tests so I'm not sure why output has changed

Reply:
My mistake, I've forgot to add `addCurrentPageToSummaryFilters` to this test case. Fixed.